src/resources/api-keys-resource.js

```diff
@@ -1,5 +1,10 @@
 export default function ApiKeysResource({apiHandler}) {
     return {
+        /**
+        * @param { rebilly.GetApiKeyCollectionRequest } data
+        * @returns { rebilly.GetApiKeyCollectionResponse } collection
+        */
+        // @ts-ignore
```

Comment:
Is it mistake?

Reply:
We need to temporary ignore these types because they are correct according to the openAPI spec but wrong according to the JS-SDK usage. 

For example, in this concrete case `sort` was present in the SDK but it is not valid according to the openAPI spec.

I checked some of those with @mclassic and we have many discrepancies that we will need to fix in following cycles.

Reply:
Tomorrow I will create a 1 week RFC for that. This cannot be automated because in some cases we are sending some parameters that are valid but not present in the openAPI spec, which is not telling us the truth because of compatibility reasons (I guess).